e2e/rook-test-framework/utils/exec_utils.go

```diff
@@ -0,0 +1,212 @@
+package utils
```

Comment:
this file looks to have some duplication as well.  Can it be refactored to remove the duplication?